### PR TITLE
Make it so each page only has a single element.

### PR DESCRIPTION
**Pull request by Kapin**
This change does a few things.

1) On Page creation, we now create an element for the page
2) Removes the ability to add multiple elements to a page on the front-end
3) Makes choices a non-required field (to make things behave)
4) CSS changes

### CODE REVIEW

File: src-frontend/app/controllers/procedure.js [hunk 1]
```diff
@@ -15,8 +15,20 @@ export default Ember.Controller.extend({
                 displayIndex: selectedIndex
             });
 
+            var controller = this;
```
Comment (Trinovantes) [inline src-frontend/app/controllers/procedure.js]: make this `procedureController`

**Comment by Ommy** [comment]
Do we still have a page has many relation with element?

**Comment by ConnorCimowsky** [comment]
LGTM

File: src-frontend/app/controllers/procedure.js (hunk 1 above)
Comment (Kapin) [inline src-frontend/app/controllers/procedure.js]: uggh ok

**Comment by Kapin** [comment]
@Ommy yes. We are just constraining it on the client right now.

**Comment by Trinovantes** [comment]
+1

**Comment by Kapin** [comment]
"ehhh LGTM" -@Ommy

**Pull request merged**